manage.py

```diff
@@ -64,11 +79,31 @@ def run(self, command):
         Args:
             command: [String] Command in self.commands.
         """
-        getattr(self, command)()  # Validation already happened in the ArgumentParser.
+        try:
+            getattr(self, command)()  # Validation already happened in the ArgumentParser.
+        except ManagerError as error:
+            # Print error message, not full stack trace.
+            sys.exit('\n{}: {}'.format(type(error).__name__, error))
+
+    def _validate_config(self):
+        """The BinaryAlert config must be well-defined before any deploy or boto3 call.
+
+        Raises:
+            InvalidConfigError: If 'aws_region' or 'name_prefix' is not defined.
+        """
+        if not self._config.get('aws_region') or not self._config.get('name_prefix'):
```

Comment:
I don't expect this should ever grow. If it does, yeah an iterable would make more sense